Taskfile.yml

```diff
@@ -294,6 +294,18 @@ tasks:
       VERBOSE:
         sh: if [ $TEST_FILTER ];  then echo "-v"; fi
 
+  # This target is intended for local use only
+  controller:test-integration-envtest-record:
```

Comment:
I think would be a good idea to add TIMEOUT setting like `{{default "15m" .TIMEOUT}}` for `controller:test-integration-envtest` and then use something like below for this target. That'll make the exiting `controller:test-integration-envtest` configurable and avoid duplication here. 

```
   cmds:
      - task: controller:test-integration-envtest
        vars: {TIMEOUT: 1h}
```

Reply:
I like this idea - nice and DRY.